### DISCUSSION TITLE
Status of vanilla-extract

### DISCUSSION THREAD

**Discussion by typeofweb** [vanilla-extract-css/vanilla-extract#1144] [General]
Hey, first of all let me tell you that I absolutely love `vanilla-extract` – the zero-runtime, the typesafety, the built-in theming, and overall great DX.

I wanted to ask about the status of this project – is `vanilla-extract` actively maintained? [Some comments in issues](https://github.com/vanilla-extract-css/vanilla-extract/pull/1105#issuecomment-1646775282) suggested it is not under development any more, hence my question.

And more importantly, do you need help with this project? Are you open to accepting PRs with bugfixes?

Thanks.

**Comment by mattcompiles** [comment]
Hey @mmiszy 👋 

I guess this depends on your definition of "maintained". We have a group of maintainers/contributors who discuss the project at least weekly. However we're very short on capacity to work on issues and new features due to family or day job commitments so progress can be slow. Issues surrounding specific setups and integrations are often the hardest for us. 

We're very open to bug fixes and as long as a fix/solution is not too complex, controversial or poorly implemented then you can expect it will be reviewed, merged and released in a reasonable amount of time.
